go.mod

```diff
@@ -253,11 +281,13 @@ require (
 	cloud.google.com/go/pubsub v1.25.1
 	github.com/davecgh/go-spew v1.1.1 // indirect
 	github.com/go-git/go-git/v5 v5.4.2
-	github.com/inconshreveable/mousetrap v1.0.0 // indirect
+	github.com/inconshreveable/mousetrap v1.0.1 // indirect
 	github.com/pmezard/go-difflib v1.0.0 // indirect
 	github.com/spf13/pflag v1.0.5 // indirect
-	golang.org/x/sys v0.0.0-20220728004956-3c1f35247d10 // indirect
-	google.golang.org/api v0.94.0
+	golang.org/x/sys v0.1.0 // indirect
+	google.golang.org/api v0.100.0
 	gopkg.in/yaml.v3 v3.0.1 // indirect
 	sigs.k8s.io/bom v0.3.0
 )
+
+replace sigs.k8s.io/release-utils v0.7.3 => github.com/cpanato/release-utils v0.3.1-0.20221106093939-c1756ce53829
```

Comment:
will remove after this get merged: https://github.com/kubernetes-sigs/release-utils/pull/69